libraries/RC_Channel/RC_Channel.h

```diff
@@ -182,6 +182,7 @@ class RC_Channel {
         Q_ASSIST =            82, // disable, enable and force Q assist
         ZIGZAG_Auto =         83, // zigzag auto switch
         AIRMODE =             84, // enable / disable airmode for copter
+        TER_DISABLE =         85, // disable terrain following in CRUISE/FBWB modes
```

Comment:
this is a dup of GENERATOR, need to use 86

Reply:
okay....GENERATOR was misplaced in the .h file  in the 100s section....why was this not caught by the tests???

Reply:
This kind of problem (with positioning of comments) would have to be caught by something like the python param metadata parser